src/Test.UnitTests.BinSkim.Rules/StringToVersionMapTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using FluentAssertions;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    public class StringToVersionMapTests
+    {
+        [Fact]
+        public void StringToVersionMap_GetVersionByKey_ShouldNotKeyNotFoundException_IfKeyDoesNotExist()
+        {
+            const string key1 = "key1";
```

Comment:
>key1 [](http://example.com/codeflow?start=25&length=4)

let''s avoid numbers in variable names.

Reply:
removed the entire class and the extension